apps/pay/components/payment-outcome/index.tsx

```diff
@@ -33,10 +33,26 @@ gql`
         __typename
       }
       status
+      paymentHash
     }
   }
 `
 
+const sendDataToPosCompanion = ({
+  username,
+  amount,
+  paymentHash,
+}: {
+  username: string
+  amount: string
+  paymentHash: string
+}) => {
+  const deepLinkUrl = `blink-pos-companion://print?username=${encodeURIComponent(
```

Comment:
we should also pass the time here. 

we should also be able to go to past transactions and print a receipt for those transactions. this would assume that time is not always `now()`. 

but approving now, we can iterate on that in follow up PRs